localisation/00_wc_vanilla_replacements.csv

```diff
@@ -496,3 +496,4 @@ EVTOPTA_MNM_1600_TT;Power and responsibility of magic is what future generations
 EVTDESC_HL_4814;Your horde has looted, ravaged, pillaged and plundered [target_location.GetName] to such a degree that a holding in the county has been completely devastated. All that is left is a smattering of rubble among our new grazing lands - our cattle will surely like it here!;Votre horde a pill� et ravag� [target_location.GetName] � un tel degr� qu'une propri�t� dans le comt� a �t� compl�tement d�vast�e. Tout ce qui reste ce sont des petits tas de d�combres au milieu de nos nouveaux p�turages tant appr�ci�s par notre b�tail!;Eure Horde hat [target_location.GetName] derart gepl�ndert, verw�stet, gebrandschatzt und ausgenommen, dass eine Liegenschaft in der Grafschaft v�llig vernichtet wurde. Nur ein Haufen Ger�ll auf unserem neuen Weideland erinnert noch daran. Unsere Vieh wird sich hier sehr wohlf�hlen!;;Tu horda ha saqueado, devastado y rapi�ado [Root.GetName] hasta tal extremo que un asentamiento en el condado ha sido completamente devastada. No quedan m�s que un pu�ado de escombros entre nuestros nuevos pastos �seguro que a nuestros caballos les gusta este lugar!;;;;;;;;;x
 alert_at_peace_prestige_loss_delayed;As a �Y$RELIGION$ Worshipper�!, you lose �R$PRESTIGE$�! Prestige every month for being at peace and doing no raiding.;En tant qu'�Yadorateur$RELIGION$�, vous perdez �R$PRESTIGE$�! Prestige chaque mois pour �tre en paix et ne pas faire de raids.;Als ein �Y$RELIGION$-Anbeter�! verliert Ihr �R$PRESTIGE$�! Prestige in jedem Monat, in dem Ihr Euch im Frieden befindet und niemanden pl�ndert.;;;;;;;;;;;x
 STAT_MARTIAL;�YState Martial�!;�YComp�tence martiale d'�tat�!;�YStaatskriegskunst�!;;�YMarcial estatal�!;;;;;;;;;x
+upgrade_stable_desc;[Root.GetGreatStableDescription];;;;;;;;;;;;;x
```

Comment:
Stables don't have a description.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/92308568-97019500-efaf-11ea-9ad5-a17a92192219.png)

</details>

Reply:
I was not able to reproduce it

Reply:
When I looked at stables, I found a description.

<details>
<summary>Image</summary>

![GOA 2](https://user-images.githubusercontent.com/44341391/92314216-72a3b980-efab-11ea-8525-5893b975dedc.png)

</details>

Reply:
> I was not able to reproduce it

Maybe on my end. I'll keep testing tomorrow.

Reply:
Works now.